react-components/table-intro/step7.md

```diff
@@ -1,98 +1,27 @@
-In this next step, we'll add the state properties we need to track how many rows per page the user can display, what current page the user is on within the dataset, and we'll add a function to handle updating the number of rows per page to display when the user changes this value with the dropdown.
+As mentioned in step 5, we can handle the case for large datasets on the interaction side, also. We can do this by incorporating pagination controls that give users more granular control over their view of the data. The Pagination component is available via Patternfly React core package `@patternfly/react-core`
```

Comment:
```suggestion
As mentioned in step 5, we can handle the case for large datasets on the interaction side. Do this by incorporating pagination controls that give users more granular control over their view of the data. The pagination component is available via patternfly React core package `@patternfly/react-core`
```

Reply:
Why lowercase `patternfly` but not `React`? I had been using uppercase for `Patternfly` and component names in general, thinking of them as nouns. If we use lowercase for these, shouldn't we do the same for `React`?

Reply:
`PatternFly` is the name of the design system, the P and F should be capitalized. @gheake2